src/Aspire.Dashboard/Components/Pages/TraceDetail.razor

```diff
@@ -52,6 +52,13 @@
                     @Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailTotalSpansHeader)] <strong>@trace.Spans.Count</strong>
                 </div>
 
+                <FluentButton Appearance="Appearance.Lightweight" OnClick="@OnCollapseAllAsync" title="@Loc[nameof(Dashboard.Resources.TraceDetail.TraceDetailCollapseAll)]">
```

Comment:
src/Aspire.Dashboard/Components/Pages/TraceDetail.razor(55,148): error CS0117: (NETCORE_ENGINEERING_TELEMETRY=Build) 'TraceDetail' does not contain a definition for 'TraceDetailCollapseAll